include/deal.II/matrix_free/fe_point_evaluation.h

```diff
@@ -1603,6 +1603,26 @@ FEPointEvaluation<n_components_, dim, spacedim, Number>::reinit(
   current_face_number = numbers::invalid_unsigned_int;
 
   do_reinit();
+
+  if (!fast_path)
+    {
+      const auto &triangulation = mapping_info->get_triangulation();
+      const auto  cell_level_and_index =
+        mapping_info->get_cell_level_and_index(cell_index);
+      const auto unit_points = mapping_info->get_unit_points_scalar(cell_index);
+
+      fe_values = std::make_shared<FEValues<dim, spacedim>>(
+        *mapping,
+        *fe,
+        Quadrature<dim>(
+          std::vector<Point<dim>>(unit_points.begin(), unit_points.end())),
+        update_flags);
```

Comment:
This will be ridiculously slow compared to the rest of the class. We should query the polynomial space directly at some point rather than use the old-fashioned `FEValues` way, but one step at a time.